src/events/order/cd-event-sessions.vue

```diff
@@ -220,12 +229,32 @@
 <style scoped lang="less">
   @import "~@coderdojo/cd-common/common/_colors";
   @import "../../common/styles/cd-primary-button";
-
+  @import "../../common/styles/cd-filler-loading";
+  .cd-filler-loading;
   .cd-event-sessions {
+    &--filler {
```

Comment:
All these styles would be the same across all usages of filler, right? Perhaps then a `cd-filler` class can just be exposed by the imported LESS file, and just add that class to the filler div?

Reply:
Fair :)